submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/SubmissionApiTest.kt

```diff
@@ -311,6 +312,39 @@ class SubmissionApiTest(
         }
     }
 
+    @Test
+    fun `16-12 User with Ftp based folder submission`() {
+        securityTestService.ensureUserRegistration(FtpSuperUser)
+        webClient = getWebClient(serverPort, FtpSuperUser)
+
+        val file = tempFolder.createFile("single.txt")
+        webClient.uploadFile(file)
+
+        val fileList = tempFolder.createFile(
+            "FileList.tsv",
+            tsv {
+                line("Files")
+                line(file.name)
+            }.toString()
+        )
+
+        webClient.uploadFile(fileList)
+
+        val submission = tsv {
+            line("Submission", "SFTP-1")
+            line("Title", "FTP user Submission")
+            line()
+
+            line("Study")
+            line("File List", "FileList.tsv")
```

Comment:
Since this test is specifically for FTP files, I would also include an inner submission file and not only file lists files, just to test all the branches of the code